### PR TITLE
Add CUHK-SZ

### PR DESCRIPTION
**Pull request by wangjiongw**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

0. Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv"). **IF YOU DO NOT DO THIS, YOUR COMMIT WILL BE SUMMARILY REJECTED.**

1. _Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

2. _Insert new faculty **in alphabetical order**, not at the end of `csrankings-[a-z].csv`._ **Do not modify `csrankings.csv`, which is auto-generated.** **DO NOT MODIFY `csrankings-[0-9].csv`, WHICH ARE BEING DEPRECATED. USE `csrankings-[a-z].csv` INSTEAD.**

3. _Read and check **all** the boxes below by filling them in with an X._

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. Faculty should also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**; please **do not** use the numeric suffixes, which are being obsoleted); include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [x] If the institution you are adding is not in the US,
update `country-info.csv`.

**(Advanced) Quick contribution via a shallow clone** 

A full clone of the CSrankings repository is almost 2GB.  To contribute a change without creating a full local clone of the CSrankings repo, you can do a _shallow clone_.  To do so, follow these steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)
2. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.
3. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to do another contribution and some time has passed, perform steps 1-3 again, creating a fresh fork and shallow clone.

### CODE REVIEW

File: csrankings-z.csv [hunk 1]
```diff
@@ -212,6 +215,7 @@ Zhiyuan Liu 0001,Tsinghua University,http://nlp.csai.tsinghua.edu.cn/~lzy,dT0v5u
 Zhiyuan Luo 0001,Royal Holloway University of London,https://pure.royalholloway.ac.uk/portal/en/persons/zhiyuan-luo_cb78d990-b108-4090-8626-afe8c14f1936.html,NOSCHOLARPAGE
 Zhiyuan Wan,Zhejiang University,https://zhiyuan-wan.github.io,e-HfEUYAAAAJ
 Zhiyun Qian,Univ. of California - Riverside,http://www.cs.ucr.edu/~zhiyunq,NOSCHOLARPAGE
+Zhizheng Wu,CUHK-SZ,http://www.drwuz.com/,K6zhweAAAAAJ
```
Comment (emeryberger) [inline csrankings-z.csv]: Web page says "Research scientist at Meta."

File: csrankings-b.csv [hunk 2]
```diff
@@ -70,7 +70,9 @@ Baoliu Ye,Nanjing University,https://cs.nju.edu.cn/yebl,BlbNI_sAAAAJ
 Baoquan Chen,Peking University,https://cfcs.pku.edu.cn/baoquan,iHWtrEAAAAAJ
 Baotian Hu,Harbin Institute of Technology,http://faculty.hitsz.edu.cn/hubaotian,5NiJ1VoAAAAJ
 Baowen Xu,Nanjing University,https://cs.nju.edu.cn/xubaowen,NOSCHOLARPAGE
+Baoxiang Wang 0001,CUHK-SZ,bxiangwang.github.io,cQe4OeYAAAAJ
```
Comment (emeryberger) [inline csrankings-b.csv]: Home page needs to be a properly formed URL.

File: csrankings-c.csv [hunk 3]
```diff
@@ -503,6 +503,7 @@ Chenshu Wu,University of Hong Kong,https://cswu.me,q9llreMAAAAJ
 Chentao Wu,Shanghai Jiao Tong University,http://www.cs.sjtu.edu.cn/~wuct,RAs-wnEAAAAJ
 Chenxi Zhang,Tongji University,https://sse.tongji.edu.cn/Data/View/2825,NOSCHOLARPAGE
 Chenyang Lu 0001,Washington University in St. Louis,http://www.cs.wustl.edu/~lu,tCq7Wx0AAAAJ
+Chenye Wu,CUHK-SZ,www.wuchenye.cn,Eivoa5IAAAAJ
```
Comment (emeryberger) [inline csrankings-c.csv]: Home page ill-formed. Please clarify relationship between SSE and SDS at CUHK. It's not clear that faculty in SSE can solely supervise PhDs in _Computer Science_.

File: csrankings-c.csv [hunk 4]
```diff
@@ -621,6 +622,7 @@ Chris Clack,University College London,http://www0.cs.ucl.ac.uk/people/C.Clack.ht
 Chris Clifton,Purdue University,https://www.cs.purdue.edu/people/clifton,C_cWJIkAAAAJ
 Chris Crispin-Bailey,University of York,https://www-users.cs.york.ac.uk/~chrisb,QQ_dWtcAAAAJ
 Chris De Sa,Cornell University,http://www.cs.cornell.edu/~cdesa,v7EjGHkAAAAJ
+Chris Ding,CUHK-SZ,https://sds.cuhk.edu.cn/teacher/197,q7FfnjgAAAAJ
 Chris Ding,Chinese University of Hong Kong,https://sds.cuhk.edu.cn/en/teacher/197,q7FfnjgAAAAJ
```
Comment (emeryberger) [inline csrankings-c.csv]: Notice there are two listings for the same faculty member. Which institution are they at?

**Review by emeryberger — COMMENTED** [review]
Please address the issue of SDS vs. SSE and how this relates to eligibility.

File: csrankings-z.csv (hunk 1 above)
Comment (wangjiongw) [reply csrankings-z.csv]: Thanks for comments. Homepage of Prof. Wu has updated to CUHKSZ

File: csrankings-c.csv (hunk 4 above)
Comment (wangjiongw) [reply csrankings-c.csv]: Prof. Chris Ding is with CUHK-SZ. I corrected the replication

File: csrankings-c.csv (hunk 3 above)
Comment (wangjiongw) [reply csrankings-c.csv]: We appreciate very much your responses to our previous request regarding listing CUHK-Shenzhen in CSRankings.
We have followed your suggestions and amended the information to be complete and accurate. Next is a summary of our clarifications and amendments.
The Chinese University of Hong Kong, Shenzhen(CUHK-Shenzhen) has two computer science and engineering (CSE) related programs (and faculty), under the School of Data Science(SDS) and School of Science and Engineering(SSE). Details can be found at the following links.
Homepage: SDS https://sds.cuhk.edu.cn/en ; SSE https://sse.cuhk.edu.cn/en
PhD program in CS area: SDS https://sds.cuhk.edu.cn/en/phd-programmes ; SSE https://sse.cuhk.edu.cn/en/taxonomy/term/157
Academic Area: SDS https://sds.cuhk.edu.cn/en/page/525 ; SSE https://sse.cuhk.edu.cn/en/page/30

File: csrankings-b.csv (hunk 2 above)
Comment (wangjiongw) [reply csrankings-b.csv]: The URL is completed in the newest modification.

**Comment by wangjiongw** [comment]
> Please address the issue of SDS vs. SSE and how this relates to eligibility.

**Pull request closed without merging**